Frameworks/UIKit/_UIGestureCoordinator.h

```diff
@@ -0,0 +1,74 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#import <UIKit/UIEvent.h>
+#import <UIKit/UITapGestureRecognizer.h>
+
+UIKIT_EXPORT_CLASS
+@interface _UIGestureCoordinator : NSObject
+
+// singleton to be used globally
++ (instancetype)singleton;
+
+- (instancetype)init;
+- (void)dealloc;
+
+// Main entrance to send touch to gesture pipleline for processing, invoked by UIView
+- (bool)processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEventName:(SEL)eventName;
+
+// Reset all gestures in all container to possible state and remove them from their container
```

Comment:
>all container to possible state [](start = 25, length = 31)

Not sure I understand this comment - all containers to possible state - what state?